### PR TITLE
AC Automated Test LRAC-9084 | master

### PR DESCRIPTION
**Pull request by 91nilo**
https://issues.liferay.com/browse/LRAC-9084

### CODE REVIEW

**Comment by 91nilo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase [hunk 1]
```diff
@@ -9,7 +9,17 @@ definition {
...
+			ACUtils.launchAC();
+
+			ACSettings.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Set up instance and sign in DXP") {
+			TestCase.setUpPortalInstance();
+
+			ACUtils.launchDXP();
+		}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase:22]: hi @91nilo , I don't think you need to connect with DXP to automate this test case...

Could you please double-check?

**Comment by 91nilo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase (hunk 1 above)
Comment (91nilo) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase]: hi @alexcandidos i fixed the setup and the test to don't connect with the dxp anymore. If there is anything else I could do to improve the code, just let me know.
Thanks!

**Comment by alexcandidos** [comment]
Hi @91nilo , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**